src/app/shared/services/app-config.service.ts

```diff
@@ -296,6 +297,25 @@ export class AppConfigService {
             priority: 1,
             order: ['label', 'title', 'authors', 'arxiv_eprint']
           },
+          '/references/items/properties/reference/properties/misc/items': {
+            onValueChange: (path, value, jsonStore, keyStore) => {
```

Comment:
We can actually put this logic into `FieldSplitterService.splitReferenceMisc`, since there is no reason to expose only a generic function from a service.

In that way we'll keep the config cleaner but a disadvantage is that person who will want to edit mappings config later will need to go in to that service. But we can always put it `AppConfigService.referenceMiscSplitterMappings` and use it in the service.

@jmartinm